### PR TITLE
fix: Add closeIcon for Tabs.Tabpane

### LINKED ISSUE #23958: 希望Tabs.TabPane增加一个API可以自定义关闭图标
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
标签页未保存时，关闭图标自定义

### What does the proposed API look like?
closeIcon     
```
      <Tabs
        onChange={this.onChange}
        activeKey={this.state.activeKey}
        type="editable-card"
        onEdit={this.onEdit}

      >
        {this.state.panes.map(pane => (
          <TabPane tab={pane.title} key={pane.key} closable={pane.closable} colseIcon={<TagOutlined />} >
            {pane.content}
          </TabPane>
        ))}
      </Tabs>
```

### PR DESCRIPTION
**Pull request by njmqd**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/23958

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Tabs support `closeIcon` of Tabpane         |
| 🇨🇳 Chinese |   Tabs.Tabpane 新增 `closeIcon` 属性支持自定义关闭图标。      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/tabs/demo/editable-card.md](https://github.com/njmqd/ant-design/blob/feature-tabs-custom-closeIcon/components/tabs/demo/editable-card.md)
[View rendered components/tabs/index.en-US.md](https://github.com/njmqd/ant-design/blob/feature-tabs-custom-closeIcon/components/tabs/index.en-US.md)
[View rendered components/tabs/index.zh-CN.md](https://github.com/njmqd/ant-design/blob/feature-tabs-custom-closeIcon/components/tabs/index.zh-CN.md)

### CODE REVIEW

File: components/tabs/index.zh-CN.md [hunk 1]
```diff
@@ -49,3 +49,4 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
 | forceRender | 被隐藏时是否渲染 DOM 结构 | boolean           | false  |
 | key         | 对应 activeKey            | string            | 无     |
 | tab         | 选项卡头显示文字          | string\|ReactNode | 无     |
+| closeIcon   | 自定义关闭图标            | ReactNode         | 无     |
```
Comment (afc163) [inline components/tabs/index.zh-CN.md:52]: 声明一下只在 editable card 类型下生效。

Comment (njmqd) [reply components/tabs/index.zh-CN.md:52]: 好的

File: components/tabs/index.zh-CN.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ Ant Design 依次提供了三级选项卡，分别用于不同的场景。
...
-| forceRender | 被隐藏时是否渲染 DOM 结构 | boolean           | false  |
-| key         | 对应 activeKey            | string            | 无     |
-| tab         | 选项卡头显示文字          | string\|ReactNode | 无     |
-| closeIcon   | 自定义关闭图标            | ReactNode         | 无     |
+| 参数        | 说明                                           | 类型              | 默认值 |
+| ----------- | ---------------------------------------------- | ----------------- | ------ |
+| forceRender | 被隐藏时是否渲染 DOM 结构                      | boolean           | false  |
+| key         | 对应 activeKey                                 | string            | 无     |
+| tab         | 选项卡头显示文字                               | string\|ReactNode | 无     |
+| closeIcon   | 自定义关闭图标，在 type="editable-card" 时有效 | ReactNode         | 无     |
```
Comment (afc163) [inline components/tabs/index.zh-CN.md:52]:
```suggestion
| closeIcon   | 自定义关闭图标，在 `type="editable-card"` 时有效 | ReactNode         | 无     |
```

**Comment by afc163** [comment]
rebase master

**Comment by afc163** [comment]
image

补充一下信息。

**Comment by afc163** [comment]
feature branch

**Comment by yoyo837** [comment]
Commits should be clean.

**Review by afc163 — APPROVED** [review]

File: components/tabs/index.zh-CN.md (hunk 2 above)
Comment (topsoft-support) [reply components/tabs/index.zh-CN.md]: 如果在一个`Tabs`里面的多个`TabPane`，需要固定几个`TabPane`不允许被关闭，其他都是可以关闭的，即：Tabs->`type="editable-card"`，但是这样 所有的`TabPane`都可以出现关闭图标，可否在每个`TabPane`中设置不现实关闭图标，类似`closeIcon={null}`这样

Comment (topsoft-support) [reply components/tabs/index.zh-CN.md]: sorry，刚看到应该在新版本中支持了，`<TabPane tab={pane.title} key={pane.key} closable={pane.closable}>`

**Pull request merged**